ui/uihandler.go

```diff
@@ -144,18 +146,18 @@ func Handler(w http.ResponseWriter, r *http.Request) {
 		flusher.Flush()
 	}
 	if DoExit {
-		syscall.Kill(syscall.Getpid(), syscall.SIGINT) // nolint: errcheck
+		syscall.Kill(syscall.Getpid(), syscall.SIGINT) // nolint: errcheck,gas
 		return
 	}
-	resolution, _ := strconv.ParseFloat(r.FormValue("r"), 64)
-	percList, _ := stats.ParsePercentiles(r.FormValue("p"))
-	qps, _ := strconv.ParseFloat(r.FormValue("qps"), 64)
+	resolution, _ := strconv.ParseFloat(r.FormValue("r"), 64) // nolint: gas
```

Comment:
what is this lint?

Reply:
it's complaining about ignoring (the _) the possible error